quickstarts/datazoom/config.yml

```diff
@@ -0,0 +1,46 @@
+# Name of the quickstart (required)
+name: datazoom-quickstart
```

Comment:
```suggestion
name: datazoom
```

Reply:
We are recommended we don't use quickstart in our names and titles as it's redundant.